BUILD.md

```diff
@@ -142,141 +121,91 @@ At this point, you can build the solution from the command line or open the
 generated solution with Visual Studio.
 
 #### Build the Solution From the Command Line
-
-While still in the build directory:
-
+To build the Debug configuration, run the following command from the build
+directory:
 ```
 cmake --build .
 ```
 
-to build the Debug configuration (the default), or:
-
+To build the Release configuration, run the following command from the build
+directory:
 ```
 cmake --build . --config Release
 ```
 
-to make a Release build.
-
 #### Build the Solution With Visual Studio
-
-Launch Visual Studio and open the "GFXReconstruct.sln" solution file
-in the build folder. You may select "Debug" or "Release" from the Solution
+Launch Visual Studio and open the `GFXReconstruct.sln` solution file
+from the build folder. You may select "Debug" or "Release" from the Solution
 Configurations drop-down list. Start a build by selecting the Build->Build
 Solution menu item.
 
-
-## Building On Linux
-
-### Linux Build Requirements
-
-This repository has been built and tested on the two most recent Ubuntu LTS
-versions. Currently, the oldest supported version is Ubuntu 14.04, meaning
-that the minimum supported compiler versions are GCC 4.8.2 and Clang 3.4,
-although earlier versions may work. It should be straightforward to adapt this
-repository to other Linux distributions.
-
+## Building for Linux
 #### Required Package List
-
 Building on Linux requires the installation of the following packages:
-
-* A C++ compiler
-* GIT
+* A C++ compiler with C++-14 support
+* Git
 * CMake
-* X11, XCB and Wayland development libraries
-
-Additional recommended tools packages include:
-
-* Python 3
+* X11 + XCB and/or Wayland development libraries
 
 ##### Ubuntu
-
 On Ubuntu, the required packages can be installed with the following
 command:
-
 ```
 sudo apt-get install git cmake build-essential libx11-xcb-dev \
         libxkbcommon-dev libwayland-dev libxrandr-dev
 ```
 
 ##### Fedora Core
-
 On Fedora Core, the required packages can be installed with the following
 command:
-
 ```
 sudo dnf install git cmake libxcb-devel libxkbcommon-devel \
         libXrandr-devel wayland-devel
 ```
 
-
 ### Linux Build
-
 The general approach is to run CMake to generate make files. Then either run
 CMake with the `--build` option or `make` to build from the command line.
 
-
 #### Use CMake to Create the Make Files
-
-Change your current directory to the top of the cloned repository directory,
-create a build directory and generate the make files.
-
+Change your current directory to the top level directory for the cloned
+GFXReconstruct repository, create a build folder, and generate the make
+files.
 ```
 cd gfxreconstruct
 mkdir build
 cmake -H. -Bbuild -DCMAKE_BUILD_TYPE=Debug
```

Comment:
It's strange to me that in the Windows description, to switch the build type the `--config` switch is used, but in the Linux description we use `-DCMAKE_BUILD_TYPE` to change the build type.

It's not a big deal; it just seems slightly self-inconsistent.

Reply:
This example is using cmake to generate build files.  The example using --config is using cmake to execute the build.